cylc/flow/network/client.py

```diff
@@ -270,7 +284,7 @@ async def async_request(
         args: Optional[Dict[str, Any]] = None,
         timeout: Optional[float] = None,
         req_meta: Optional[Dict[str, Any]] = None
-    ) -> object:
+    ) -> Union[bytes, object]:
```

Comment:
(object was used as a catch-all back before mypy)

```python
>>> isinstance(bytes, object)
True
```

```suggestion
    ) -> object:
```